src/cli_plugin/install/install.js

```diff
@@ -62,7 +63,7 @@ export async function install(settings, logger) {
 
     assertVersion(settings);
 
-    const targetDir = path.join(settings.pluginDir, settings.plugins[0].id);
+    const targetDir = path.join(settings.pluginDir, kebabCase(settings.plugins[0].id));
```

Comment:
@VachaShah  does this have any co-relation with removing plugin as well ? https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/cli_plugin/remove/remove.js

Reply:
> @VachaShah does this have any co-relation with removing plugin as well ? https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/cli_plugin/remove/remove.js

This would not affect the removing plugin logic as it does not take into consideration the plugin id.